### DISCUSSION TITLE
Issue with font rendering

### DISCUSSION THREAD

**Discussion by roland-5** [wezterm/wezterm#4094] [Q&A]
I wanted to migrate from Alacritty, but have little problem with difference how the same font looks in Wezterm and in Alacritty. 
Normally, I use `MesloLGS Nerd Font` as my main font - maybe there is something better, but I tried many fonts and there were always some problems for my brain and fonts for Dyslexia didn't help me.

`e`, `t` or `i` looks more... bulky?  In Alacritty they are more slim.  Although they have the same height, in the case of Alacritta they seem higher to my brain and therefore easier to read for me.

Can I somehow influence how Wezterm displays this font or is it a matter of its rendering engine and nothing can be done about it?
 
```
-- Pull in the wezterm API
local wezterm = require("wezterm")

-- This table will hold the configuration.
local config = {}

-- In newer versions of wezterm, use the config_builder which will
-- help provide clearer error messages
if wezterm.config_builder then
	config = wezterm.config_builder()
end

-- This is where you actually apply your config choices
-- For example, changing the color scheme:
config.color_scheme = "GruvboxDarkHard"

-- Set background to same color as neovim
config.colors = {}
config.colors.background = "#000000"

config.font = wezterm.font("MesloLGS Nerd Font")
config.font_size = 17.3

-- config.font = wezterm.font_with_fallback {
--   'Berkeley Mono',
--   'nonicons',
-- }

config.enable_tab_bar = false

-- I don't like putting anything at the edge if I can help it.
config.enable_scroll_bar = false
config.window_padding = {
	left = 0,
	right = 0,
	top = 0,
	bottom = 0,
}

-- Mouse cursor
config.default_cursor_style = "BlinkingBlock"
config.cursor_blink_rate = 900

config.check_for_updates = false

config.adjust_window_size_when_changing_font_size = false

config.freetype_load_target = "HorizontalLcd"

-- and finally, return the configuration to wezterm
return config
```

**Comment by roland-5** [comment]
Alacritty
![ps_20230804182546](https://github.com/wez/wezterm/assets/44477799/0af89a81-8351-4fef-b2cf-a99dd472da3c)

**Comment by roland-5** [comment]
Wezterm
![ps_20230804182551](https://github.com/wez/wezterm/assets/44477799/eca37e0e-29f5-4273-b631-0e70a13764e0)

**Comment by cultab** [comment]
You could try using a lighter font weight, but as someone that had the same issue I know it's not the same 😞
